test/e2e/performanceprofile/functests/7_performance_kubelet_node/cgroups.go

```diff
@@ -95,3 +525,147 @@ func cpuSpecToString(cpus *performancev2.CPU) string {
 	}
 	return sb.String()
 }
+
+// deleteTestPod removes guaranteed pod
+func deleteTestPod(testpod *corev1.Pod) {
+	// it possible that the pod already was deleted as part of the test, in this case we want to skip teardown
+	err := testclient.Client.Get(context.TODO(), client.ObjectKeyFromObject(testpod), testpod)
+	if errors.IsNotFound(err) {
+		return
+	}
+
+	err = testclient.Client.Delete(context.TODO(), testpod)
+	Expect(err).ToNot(HaveOccurred())
+
+	err = pods.WaitForDeletion(testpod, pods.DefaultDeletionTimeout*time.Second)
+	Expect(err).ToNot(HaveOccurred())
+}
+
+// getOvnPod Get OVN Kubenode pods running on the worker cnf node
+func getOvnPod(workerNode *corev1.Node) (corev1.Pod, error) {
+	var ovnKubeNodePod corev1.Pod
+	ovnpods := &corev1.PodList{}
+	options := &client.ListOptions{
+		Namespace: "openshift-ovn-kubernetes",
+	}
+	err := testclient.Client.List(context.TODO(), ovnpods, options)
```

Comment:
same, first parameter of the function should be `ctx context` and it should be passed to all fucntions which accepts `context`

Reply:
Please change the `getOvnPod` to get `context` as it first argument and pass it to the `Client.List` call

Reply:
Addressed in the latest commit